### ISSUE TITLE
PEP 3134 -- Exception Chaining and Embedded Tracebacks

### ISSUE THREAD

**Issue by lafrank** [IronLanguages/ironpython3#70]
Although I can compile the code, it seems tokenizer does not understand the from statement in a raise clause as per PEP3134.

A dummy sample code :

``` python
  try:
    pass
  except Exception as e:
    raise NameError from e

IronPython throws syntax error: 

  File "", line 5
        raise NameError from e
                        ^
SyntaxError: unexpected token 'from'
```

**Comment by simplicbe** [comment]
Does this belongs to the following issue: https://github.com/IronLanguages/ironpython3/issues/3

**Comment by lafrank** [comment]
Yes it seems the same, I did not notice it has already been raised.

**Comment by simplicbe** [comment]
Yep, we all try to to push project forward. Any help is very appreciated, so thank you!

**Comment by simplicbe** [comment]
Ok, got a first running version. If your intrested: https://github.com/IronLanguages/ironpython3/compare/f_exception_chain

**Comment by slide** [comment]
Fixed via #132
